dev-lang/perl/perl-5.32.1.recipe

```diff
@@ -16,59 +16,65 @@ HOMEPAGE="https://www.perl.org/"
-	cmd:perldoc$secondaryArchSuffix = $portVersion compat >= 5
-	cmd:perlivp$secondaryArchSuffix = $portVersion compat >= 5
-	cmd:perlthanks$secondaryArchSuffix = $portVersion compat >= 5
-	cmd:piconv$secondaryArchSuffix = $portVersion compat >= 5
-	cmd:pl2pm$secondaryArchSuffix = $portVersion compat >= 5
-	cmd:pod2html$secondaryArchSuffix = $portVersion compat >= 5
-	cmd:pod2man$secondaryArchSuffix = $portVersion compat >= 5
-	cmd:pod2text$secondaryArchSuffix = $portVersion compat >= 5
-	cmd:pod2usage$secondaryArchSuffix = $portVersion compat >= 5
-	cmd:podchecker$secondaryArchSuffix = $portVersion compat >= 5
-	cmd:podselect$secondaryArchSuffix = $portVersion compat >= 5
-	cmd:prove$secondaryArchSuffix = $portVersion compat >= 5
-	cmd:ptar$secondaryArchSuffix = $portVersion compat >= 5
-	cmd:ptardiff$secondaryArchSuffix = $portVersion compat >= 5
-	cmd:ptargrep$secondaryArchSuffix = $portVersion compat >= 5
-	cmd:shasum$secondaryArchSuffix = $portVersion compat >= 5
-	cmd:splain$secondaryArchSuffix = $portVersion compat >= 5
-	cmd:streamzip$secondaryArchSuffix = $portVersion compat >= 5
-	cmd:xsubpp$secondaryArchSuffix = $portVersion compat >= 5
-	cmd:zipdetails$secondaryArchSuffix = $portVersion compat >= 5
+	cmd:corelist = $portVersion compat >= 5
+	cmd:cpan = $portVersion compat >= 5
+	cmd:enc2xs = $portVersion compat >= 5
+	cmd:encguess = $portVersion compat >= 5
+	cmd:h2ph = $portVersion compat >= 5
+	cmd:h2xs = $portVersion compat >= 5
+	cmd:instmodsh = $portVersion compat >= 5
+	cmd:json_pp = $portVersion compat >= 5
+	cmd:libnetcfg = $portVersion compat >= 5
+	cmd:perl = $portVersion compat >= 5
+	cmd:perl$portVersion = $portVersion compat >= 5
+	cmd:perlbug = $portVersion compat >= 5
+	cmd:perldoc = $portVersion compat >= 5
+	cmd:perlivp = $portVersion compat >= 5
+	cmd:perlthanks = $portVersion compat >= 5
+	cmd:piconv = $portVersion compat >= 5
+	cmd:pl2pm = $portVersion compat >= 5
+	cmd:pod2html = $portVersion compat >= 5
+	cmd:pod2man = $portVersion compat >= 5
+	cmd:pod2text = $portVersion compat >= 5
+	cmd:pod2usage = $portVersion compat >= 5
+	cmd:podchecker = $portVersion compat >= 5
+	cmd:podselect = $portVersion compat >= 5
+	cmd:prove = $portVersion compat >= 5
+	cmd:ptar = $portVersion compat >= 5
+	cmd:ptardiff = $portVersion compat >= 5
+	cmd:ptargrep = $portVersion compat >= 5
+	cmd:shasum = $portVersion compat >= 5
+	cmd:splain = $portVersion compat >= 5
+	cmd:streamzip = $portVersion compat >= 5
+	cmd:xsubpp = $portVersion compat >= 5
+	cmd:zipdetails = $portVersion compat >= 5
 	# vendor_perl refers to the path to packaged perl modules, which includes
 	# the short perl version, so it is backwards compatible up this one.
-	vendor_perl$secondaryArchSuffix = $perlShortVersion compat >= $perlShortVersion
-	lib:libperl$secondaryArchSuffix = $portVersion compat >= $perlShortVersion
+	vendor_perl = $perlShortVersion compat >= $perlShortVersion
+	lib:libperl = $portVersion compat >= $perlShortVersion
```

Comment:
This one is a bit of a special case of a library, as it is a library built with gcc 13 but without $secondaryArchSuffix because it isn't in $libDir. But there is also no devel counterpart for it, so it is a bit special anyway.

Reply:
fixed in 14c4afa